doc/sphinx/source/quickstart/installation.rst

```diff
@@ -633,10 +489,9 @@ To check that the installation was successful, run
 this should show the directory of the source code that you just downloaded.
 
 If the command above shows a directory inside your conda environment instead,
-e.g. ``~/mamba/envs/esmvaltool/lib/python3.8/site-packages/esmvalcore``, you
-may need to manually remove that directory and run
-```pip install -e '.[develop]'``
-again.
+e.g. ``~/mambaforge/envs/esmvaltool/lib/python3.8/site-packages/esmvalcore``,
+you may need to manually remove that directory and run
```

Comment:
ooh no no - don't tell users to remove system dirs manually - that can lead to fireworks later, pip install is always overriding a conda install, so we should be fine here

Reply:
It has happened to me on several occasions that pip did not overwrite the conda installed package.

Reply:
your pip was broken, it always pips conda, excuse the pun :grin: Ok "you may" does it for me then, as an optional - should be good, man